unsafe/src/main/java/org/apache/spark/unsafe/array/ByteArrayMethods.java

```diff
@@ -38,18 +38,24 @@ public static int roundNumberOfBytesToNearestWord(int numBytes) {
    * Optimized byte array equality check for 8-byte-word-aligned byte arrays.
```

Comment:
This comment is now out-of-date and should be updated to reflect the fact that the new method does not require the input arrays' lengths to be multiples of 8 bytes.